### ISSUE TITLE
browser shows all entities with that term, not just those that map to anchor entity

### ISSUE THREAD

**Issue by bradfordcondon** [statonlab/tripal_cv_xray#27]
http://160.36.205.61:8095/organism/Fraxinus/excelsior?tripal_pane=group_ontologies

there are 24 'sexual reproduction' records (+1 from a child term).

<img width="684" alt="screen shot 2018-05-11 at 3 42 52 pm" src="https://user-images.githubusercontent.com/7063154/39943802-2fcab4ee-5532-11e8-8a15-d32ee6c395ef.png">

```
select F.uniquename FROM chado.feature F
INNER JOIN chado.feature_cvterm FC ON FC.feature_id = F.feature_id
INNER JOIN chado.cvterm C ON FC.cvterm_id = C.cvterm_id
WHERE C.name = 'sexual reproduction';
``` 
this has 24 records
```
FRAEX38873_v2_000064050.1
FRAEX38873_v2_000232540.1
FRAEX38873_v2_000008220.1
FRAEX38873_v2_000161920.1
FRAEX38873_v2_000235380.1
FRAEX38873_v2_000027030.1
FRAEX38873_v2_000382030.1
FRAEX38873_v2_000383640.1
FRAEX38873_v2_000038400.1
FRAEX38873_v2_000375130.1
QlobV1.0_scaffold1599_0000090-RA
QlobV1.0_scaffold1599_0000050-RA
QlobV1.0_scaffold1599_0000070-RA
QlobV1.0_C390613_0000010-RA
QlobV1.0_C390613_0000030-RA
QlobV1.0_scaffold2470_0000040-RA
QlobV1.0_scaffold9085_0000050-RA
QlobV1.0_scaffold9085_0000040-RA
QlobV1.0_scaffold4048_0000040-RA
QlobV1.0_scaffold11844_0000020-RA
QlobV1.0_scaffold6219_0000030-RA
QlobV1.0_scaffold5269_0000120-RA
QlobV1.0_scaffold5269_0000320-RA
QlobV1.0_scaffold8298_0000050-RA

```

So its getting both the Q. lobata and F. excelsior results, not just F excelsior.

**Comment by bradfordcondon** [comment]
interesitngly there are 28 records in the index.


cvterm_entity_id | cvterm_id | database | accession | entity_id
-- | -- | -- | -- | --
Edit | Delete | 51545723 | 55640 | GO | 0019953 | 294348
Edit | Delete | 51308482 | 55640 | GO | 0019953 | 289330
Edit | Delete | 51308481 | 55640 | GO | 0019953 | 289330
Edit | Delete | 51308378 | 55640 | GO | 0019953 | 289330
Edit | Delete | 51308377 | 55640 | GO | 0019953 | 289330
Edit | Delete | 51121150 | 55640 | GO | 0019953 | 285008
Edit | Delete | 50915500 | 55640 | GO | 0019953 | 271517
Edit | Delete | 50865405 | 55640 | GO | 0019953 | 270460
Edit | Delete | 50864216 | 55640 | GO | 0019953 | 270432
Edit | Delete | 50823673 | 55640 | GO | 0019953 | 269555
Edit | Delete | 50735566 | 55640 | GO | 0019953 | 267693
Edit | Delete | 50733517 | 55640 | GO | 0019953 | 267659
Edit | Delete | 50287559 | 55640 | GO | 0019953 | 248964
Edit | Delete | 50080046 | 55640 | GO | 0019953 | 244716
Edit | Delete | 49304899 | 55640 | GO | 0019953 | 218851
Edit | Delete | 49280495 | 55640 | GO | 0019953 | 218177
Edit | Delete | 49030269 | 55640 | GO | 0019953 | 204334
Edit | Delete | 48890323 | 55640 | GO | 0019953 | 201297
Edit | Delete | 48717534 | 55640 | GO | 0019953 | 196991
Edit | Delete | 48322480 | 55640 | GO | 0019953 | 180446
Edit | Delete | 48288754 | 55640 | GO | 0019953 | 179773
Edit | Delete | 47653886 | 55640 | GO | 0019953 | 156885
Edit | Delete | 47505036 | 55640 | GO | 0019953 | 144569
Edit | Delete | 47233370 | 55640 | GO | 0019953 | 138910
Edit | Delete | 47175172 | 55640 | GO | 0019953 | 137627
Edit | Delete | 47144060 | 55640 | GO | 0019953 | 137032
Edit | Delete | 47037629 | 55640 | GO | 0019953 | 134398
Edit | Delete | 46967637 | 55640 | GO | 0019953 | 132722

**Comment by almasaeed2010** [comment]
Let's verify what the entity object is in line https://github.com/statonlab/tripal_cv_xray/blob/reimplement-field/includes/api/tripal_cv_xray.api.inc#L66

**Comment by bradfordcondon** [comment]
looks reasonable enough:

<img width="913" alt="screen shot 2018-05-11 at 4 12 34 pm" src="https://user-images.githubusercontent.com/7063154/39944839-22fb6f70-5536-11e8-9e87-a0c443638abd.png">

**Comment by bradfordcondon** [comment]
compare 
http://160.36.205.61:8095/organism/Quercus/lobata?tripal_pane=group_ontologies
and 
http://160.36.205.61:8095/organism/Fraxinus/excelsior?tripal_pane=group_ontologies

child terms are same, root terms ahve diff counts.  ibet the ajax callback is wrong/needs to be updated to new api?  can it be that easy?

**Comment by almasaeed2010** [comment]
Yep it looks like we just didn't update the ajax call function. I am submitting a fix in `reimplement-field` branch.

**Comment by almasaeed2010** [comment]
Doesn't seem to have fixed the issue. I might be messing with the wrong function though.

**Comment by almasaeed2010** [comment]
Ok fixed in reimplement-field. It was just a typo in the ajax call 😅
